src/pyOpenMS/pxds/LevMarqFitter1D.pxd

```diff
@@ -6,7 +6,7 @@ cdef extern from "<OpenMS/FEATUREFINDER/LevMarqFitter1D.h>" namespace "OpenMS":
         # wrap-ignore
         # no-pxd-import
         # wrap-doc:
-        # Abstract class for 1D-model fitter using Levenberg-Marquardt algorithm for parameter optimization
+        #  Abstract class for 1D-model fitter using Levenberg-Marquardt algorithm for parameter optimization
```

Comment:
anything special about this indentation?

Reply:
same as above